contrib/demo/startKcpAndClusterController.sh

```diff
@@ -11,8 +11,13 @@ KUBECONFIG=${KCP_DATA_ROOT}/.kcp/data/admin.kubeconfig
 source ${DEMO_ROOT}/.startUtils
 setupTraps $0 "rm -Rf ${CURRENT_DIR}/.kcp.running"
 
+if [ -z "${KCP_LISTEN_ADDR}" ]; then
+    echo -e "KCP cannot be started, Please set Address:port for kcp using KCP_LISTEN_ADDR environment";
+    exit 1
```

Comment:
Why exiting here ?
I'd rather simply *not add* the `--listen` argument if the env variable is not there...

Reply:
I like your approach